src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -57,10 +57,20 @@ def validate_bicep_target_scope(template_schema, deployment_scope):
         )
 
 
-def run_bicep_command(args, auto_install=True, check_version=True):
+def run_bicep_command(cli_ctx, args, auto_install=True):
+    if _use_binary_from_path(cli_ctx):
+        from shutil import which
+
+        if which("bicep") is None:
```

Comment:
Is it worth validating here that this can be executed (and also logging the version info to debug output)?

Reply:
`which` ensures the specified file is an executable. Added logging for the Bicep version info.